docs/3.0rc/concepts/flows.mdx

```diff
@@ -1141,18 +1141,6 @@ While the cancellation process is robust, there are a few issues than can occur:
 - If the `infrastructre_pid` is missing from the flow run will be marked as cancelled but cancellation cannot be enforced.
 - If the worker runs into an unexpected error during cancellation the flow run may or may not be cancelled depending on where the error occurred. The worker will try again to cancel the flow run. Another worker may attempt cancellation.
 
-!!! tip "Enhanced cancellation"
-    We are working on improving cases where cancellation can fail.
-    You can try the improved cancellation experience by enabling the `PREFECT_EXPERIMENTAL_ENABLE_ENHANCED_CANCELLATION` setting on your worker or agents:
-
-    <div class="terminal">
-    ```bash
-    prefect config set PREFECT_EXPERIMENTAL_ENABLE_ENHANCED_CANCELLATION=True
-    ```
-    </div>
-
-    If you encounter any issues, please let us know in [Slack](https://www.prefect.io/slack/) or with a [Github](https://github.com/PrefectHQ/prefect) issue.
-
```

Comment:
enabled by default now so instead of removing the word `agent` I just removed the whole tip